storage/repair.go

```diff
@@ -184,7 +184,7 @@ type repairState struct {
 
 type namespaceRepairStateByTime map[xtime.UnixNano]repairState
 
-type repairStatesByNs map[ident.Hash]namespaceRepairStateByTime
+type repairStatesByNs map[string]namespaceRepairStateByTime
```

Comment:
How come you're not using a generated map here?

Reply:
Yeah i'm fine with leaving as is. Mind putting a quick note explaining this rationale above the type.

Reply:
Sure thing.